### DISCUSSION TITLE
Undefined references in lapack-reference during building

### DISCUSSION THREAD

**Discussion by EmielBoss** [microsoft/vcpkg#27475] [Q&A]
I use CMake to generate project files, which succeeds without error. However, when I build my project, I am getting lots of undefined references of the form:

```
[100%] Linking CXX executable sne_cmd
/usr/bin/ld: vcpkg_installed/x64-linux/debug/lib/liblapack.a(slarfb.f.o): in function `slarfb_':
<repo>/third_party/vcpkg/buildtrees/lapack-reference/src/v3.10.0-a3af633c89.clean/SRC/slarfb.f:259: undefined reference to `scopy_'
/usr/bin/ld: <repo>/third_party/vcpkg/buildtrees/lapack-reference/src/v3.10.0-a3af633c89.clean/SRC/slarfb.f:265: undefined reference to `strmm_'
/usr/bin/ld: <repo>//third_party/vcpkg/buildtrees/lapack-reference/src/v3.10.0-a3af633c89.clean/SRC/slarfb.f:278: undefined reference to `strmm_'
/usr/bin/ld: <repo>/third_party/vcpkg/buildtrees/lapack-reference/src/v3.10.0-a3af633c89.clean/SRC/slarfb.f:294: undefined reference to `strmm_'
/usr/bin/ld: <repo>/third_party/vcpkg/buildtrees/lapack-reference/src/v3.10.0-a3af633c89.clean/SRC/slarfb.f:313: undefined reference to `scopy_'
...
```

I believe this issue arose after I deleted my `build/` folder, which I've done many times before. However, I also get this issue when cloning and building any version of [the repository I forked from](https://github.com/Markvanderuit/dual_hierarchy_tsne), which used to work before as well. I tried clean clones of older versions of both my and the forked-from repository, I tried downgrading vcpkg to versions that worked before, and I tried explicitly including `lapack-reference` in my `vcpkg.json` (I'm not very good at vcpkg, so I don't know if that makes any sense). I always get these errors. I assume it's a system-wide thing?

Any help is much appreciated!

**Comment by JackBoosY** [comment]
Can you please open an issue to report that?
